package.json

```diff
@@ -11,7 +11,7 @@
     "generate": "nuxt generate"
   },
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=14.17.0"
```

Comment:
why do you need to downgrade node version?

Reply:
I'm running on 14.17 :D

Reply:
Could you change node version un DockerFile too?